spec/features/welcome/index_spec.rb

```diff
@@ -1,15 +0,0 @@
-require 'rails_helper'
```

Comment:
why is this deleted?

Reply:
we didn't need it because the only thing that the test was checking was the login by google button which we removed.

Reply:
ok